dm/dm/master/openapi_view.go

```diff
@@ -72,7 +69,7 @@ func (s *Server) InitOpenAPIHandles() error {
 	}
 	// disables swagger server name validation. it seems to work poorly
 	swagger.Servers = nil
-
+	gin.SetMode(gin.ReleaseMode)
```

Comment:
what's `view` in the filename `openapi_view.go`?

Reply:
our api is openapi,

Reply:
https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller

Reply:
[MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) mostly used with UI, but ours's openapi

Reply:
yes, but MVC has been widely adopted as a design for World Wide Web applications in major programming languages, 
you can read the rest of wiki [Use in web applications], and openapi is something like `UI` or `Web`

the advantage of using this model is that the code will be more logically layered and easier to maintain (given that the openapi codebase is already quite large)